conf.yaml

```diff
@@ -1440,6 +1440,32 @@ contents:
                         repo:   apm-agent-rum-js
                         path:   CHANGELOG.asciidoc
                         exclude_branches:   [ 3.x, 2.x, 1.x, 0.x ]
+              - title:      APM AWS Lambda extension
```

Comment:
I don't have strong feelings about this, but I think we use title caps for all the other book names.

Reply:
Ah. We do. Thanks.